runtime/default.policy

```diff
@@ -31,6 +58,53 @@ macros:
 # Rules determine what the Datadog security-agent will monitor.#
 ################################################################
 rules:
+## Process Activity Rules ##
+  - id: potential_web_shell
+    description: Potential webshell execution
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in HTTP_UTILS || exec.filename in SHELL_UTILS) && (process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*")
```

Comment:
What is the purpose of `process.ancestors.name =~ "php*" `?

Reply:
different PHP versions have different executable names (unfortunately)